exasol_script_languages_container_ci_setup/templates/buildspec_batch_entry.yaml

```diff
@@ -4,4 +4,4 @@
           FLAVOR: {{flavor_original}}
       buildspec: {{out_path}}/build_buildspec.yaml
       privileged-mode: true
-      type: BUILD_GENERAL1_MEDIUM
+      type: BUILD_GENERAL1_MEDIUM
```

Comment:
trailing newline?

Reply:
In this case I would prefer to keep it without a trailing new line, as this will make the resulting buildspec quite large.
This is one entry in a list which will be assembled.